applications/plugins/SofaPython/Binding_Data.cpp

```diff
@@ -110,31 +112,34 @@ PyObject *GetDataValuePython(BaseData* data)
     const AbstractTypeInfo *typeinfo = data->getValueTypeInfo();
     const void* valueVoidPtr = data->getValueVoidPtr();
 
-    if (!typeinfo->Container())
+    if (!typeinfo->Container() || typeinfo->ValidInfo())
```

Comment:
It's from an old change on our end (December 2014). But the original commit only added `if(typeinfo->ValidInfo())` around all the codes using `typeinfo`, `!typeinfo->Container()` was not there. So this is a merge error.

Given that this code is now completely changed in our version (with a redesign of `DataTypeInfo`), and probably conflicts with other changes in the master, I think this commit could simply be removed from this PR.